packages/cli/test/functional/test_site_templates/test_portfolio/expected/index.html

```diff
@@ -0,0 +1,260 @@
+          </panel>
+          <br>
+          <panel expanded><template #header>
+              <p>Example using icons</p>
+            </template>
+            <box type="warning">
+              Some icons are not available in the libraries MarkBind supports! Please ensure that
+              the desired icons are available before choosing this option. For more information regarding
+              icons, please refer to <a href="https://markbind.org/userGuide/formattingContents.html#icons" target="_blank">this section of our User Guide</a></box>
+            <i class="fa-brands fa-square-js fa-2xl"></i>
+            <i class="fa-brands fa-vuejs fa-2xl"></i>
+            <i class="fa-brands fa-java fa-2xl"></i>
+          </panel>
+        </div>
+        <hr>
+        <div>
+          <h1 id="experience">Experience<a class="fa fa-anchor" href="#experience" onclick="event.stopPropagation()"></a></h1>
+          <panel type="seamless" no-close minimized>
+            <template #header>
+              <div>
+                <span><strong>Software Engineering Intern @ <a href="https://nus.edu.sg/">NUS</a></strong> | May 2024 - Aug 2024</span>
+              </div>
+            </template>
+            <div class="container">
+              Role description
+            </div>
+            <div class="container">
+              <thumbnail src="/contents/assets/Vue.svg" size="50"></thumbnail>
+            </div>
+          </panel>
+          <br>
+          <panel type="seamless" no-close minimized>
+            <template #header>
+              <div>
+                <span><strong>Another experience @ <a href>Another Company</a></strong> | May 2022 - Aug 2022</span>
+              </div>
+            </template>
+            <div class="container">
+              Role description
+            </div>
+            <div class="container">
+              <thumbnail src="/contents/assets/JavaScript.svg" size="50"></thumbnail>
+            </div>
+          </panel>
+        </div>
+        <hr>
+        <div>
+          <h1 id="projects">Projects<a class="fa fa-anchor" href="#projects" onclick="event.stopPropagation()"></a></h1>
+          <div>
+            <div class="card mb-3">
+              <h3 class="card-header" id="example-project-ip-for-cs2103">Example project: iP for CS2103<a class="fa fa-anchor" href="#example-project-ip-for-cs2103" onclick="event.stopPropagation()"></a></h3>
+              <div class="card-body">
+                <span>This would be a good place to add images and embeds showcasing your contributions to a project. In this case, an <code class="hljs inline no-lang" v-pre>&lt;iframe&gt;</code> containing the <a href="https://reposense.org/index.html">RepoSense</a> report of a student's contributions to their iP is embedded. For more information on embeds, refer to our user guide <a href="/contents/(https:/markbind.org/userGuide/formattingContents.html#embeds)">here.</a></span>
```

Comment:
Link here is broken

Reply:
Yeah good catch! I've fixed it since, you can test it out here: https://portfolio-template-markbind.netlify.app/

Reply:
Hi this is still broken

Reply:
@jingting1412 I think you have to fix the broken link in the actual file instead of the expected file

Reply:
Yep sorry 😔 it should be fixed now